### PR TITLE
Add script to check OnSeekMediaClockTimer with fake parameter in seek…

### PR DESCRIPTION
**Pull request by OVikhrov**
…Time structure
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua [hunk 1]
```diff
@@ -0,0 +1,103 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [HMI RPC validation]: SDL behavior: HMI sends request (response, notification) with fake parameters that SDL should transfer to mobile
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua]: @OVikhrov, please update Requirement summary also with related to OnSeekMediaClockTimer functional requirement

File: test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {fakeparameter = "fake", hours = 1, minutes = 1, seconds = 59}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua]: This is  Functional Requirement

File: test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**